src/_shared/scss/_core.scss

```diff
@@ -10,9 +10,17 @@ $gs-column-width: 60px;
 $gs-row-height: 36px;
 $gs-max-columns: 16;
 
+// For popups.
```

Comment:
Following earlier comments about popup and branding styling, are these additional styles needed?